entrypoint

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+set -euo pipefail
+
+exec ./linksharing $@
```

Comment:
At that point, why not set linksharing as the entrypoint_

Reply:
I kept it the same way as our gateway image currently.
I will add some additional settings to this file likely in another PR after this is merged.